collection-scripts/gather

```diff
@@ -3,34 +3,12 @@
 # Resource List
 resources=()
 
-# Operator Resources
-resources+=(clusteroperators)
-
-# Certificate Resources
-resources+=(certificatesigningrequests)
-
-# Machine/Node Resources
-resources+=(nodes machines machineconfigs machineconfigpools)
-
-# Namespaces/Project Resources
-resources+=(ns/default ns/openshift ns/kube-system)
-
-# Storage Resources
-resources+=(storageclasses persistentvolumes volumeattachments)
-
-# Networking Resources
-resources+=(clusternetworks hostsubnets)
-
-# Autoscaler Resources
-resources+=(clusterautoscaler machineautoscaler)
+# Add general resources to list if necessary 
 
 # Run the Collection of Resources using must-gather
```

Comment:
is this loop still required since the resources are empty here, I was thinking it would be better if we directly call gather script for both noobaa and ceph ?

Reply:
can be done in another PR merging it for now.